ranker/__init__.py

```diff
@@ -3,6 +3,7 @@
 import csh_ldap
 from flask import Flask
 from flask_migrate import Migrate
+from flask_pyodic.provider_configuration import *
```

Comment:
Remove universal import.  you only need to include ProviderConfiguration and ClientMetadata